plugwise/helper.py

```diff
@@ -437,7 +438,9 @@ def _get_module_data(
                 model_data["firmware_version"] = module.find("firmware_version").text
                 mac_locator = ".//protocols/zig_bee_node/mac_address"
                 if module.findall(mac_locator):
```

Comment:
Why findall? A single find should suffice?

Reply:
There is also one in `smile.py`-> same as last night (which we resolved this morning) but if single find doesn't work. Not sure why